lib/rails_multitenant/global_context_registry.rb

```diff
@@ -175,10 +175,10 @@ def with_isolated_registry(registry = {})
     # Note: these methods are intended for use in a manner like .with_isolated_registry,
     # but in contexts where around semantics are not allowed.
 
-    # Set a new, empty registry, returning the previous one.
-    def new_registry
+    # Set a new, by default empty registry, returning the previous one.
+    def new_registry(registry = {})
```

Comment:
Should we update `with_isolated_registry` to use `new_registry` in its implementation?